bcl/src/bc_cp201t.c

```diff
@@ -0,0 +1,313 @@
+#include <bc_adc.h>
+#include <bc_cp201t.h>
+
+typedef struct
+{
+    bc_module_sensor_channel_t channel;
+    bc_adc_channel_t channel_adc;
+    bc_scheduler_task_id_t task_id_interval;
+    bc_scheduler_task_id_t task_id_measure;
+    void (*event_handler)(bc_module_sensor_channel_t, bc_cp201t_event_t, void *);
+    void *event_param;
+    int32_t thermistor_data;
+    float temperature;
+    bc_tick_t update_interval;
+    bool initialized;
+    bool measurement_active;
+
+} bc_cp201t_t;
+
+static bc_cp201t_t _bc_cp201t[2] =
+{
+    [0] =
+    {
+        .channel = BC_MODULE_SENSOR_CHANNEL_A,
+        .channel_adc = BC_ADC_CHANNEL_A4,
+        .initialized = false,
+    },
+
+    [1] =
+    {
+        .channel = BC_MODULE_SENSOR_CHANNEL_B,
+        .channel_adc = BC_ADC_CHANNEL_A5,
+        .initialized = false,
+    }
+};
+
+static const uint16_t _thermistor_conversion_table[1024];
```

Comment:
Correct name starts with the module name prefix - e.g.` _cp201t_conversion_table`